activerecord/lib/active_record/core.rb

```diff
@@ -96,6 +96,15 @@ def self.error_on_ignored_order_or_limit=(value)
         self.error_on_ignored_order = value
       end
 
+      # :singleton-method:
+      # Specify the behavior for unsafe raw query methods. Values are as follows
+      #   enabled    - Unsafe raw SQL can be passed to query methods.
```

Comment:
If we're planning to remove support for this in 6.0, why do we need an `:enabled` setting?